### PR TITLE
Added the ability to rerun the install script again, without raising …

### PR DESCRIPTION
**Pull request by Shimwell**
Added a few checks to see if folders exist prior to performing a git clone
Also found that the nosetests was failing so replaced it with ./travis-run-tests.sh

### CODE REVIEW

File: ubuntu_15.04.sh [hunk 1]
```diff
@@ -50,6 +53,9 @@ cd PyTAPS-1.4/
 python setup.py --iMesh-path=$HOME/opt/moab --without-iRel --without-iGeom install --user
 cd ..
 # Install PyNE
+if [ -d pyne ] ; then
+    rm -rf pyne
+fi
```
Comment (kkiesling) [inline ubuntu_15.04.sh]: Perhaps a warning should be given before deleting the pyne folder (or any folder). While it is assumed to be a clean machine, it is best to not accidentally delete a folder that may contain other things that just happens to have the same name. Perhaps having an interactive warning message to ask them if they want to proceed?

File: ubuntu_15.04.sh [hunk 2]
```diff
@@ -20,17 +20,20 @@ export LD_LIBRARY_PATH=/usr/lib/x86_64-linux-gnu
...
-mkdir opt
+mkdir -p opt
 cd opt
 # Install MOAB
-mkdir moab
+mkdir -p moab
 cd moab
+if [ -d moab ] ; then
+    rm -rf moab
+fi
```
Comment (kkiesling) [inline ubuntu_15.04.sh]: see comment below about deleting PyNE folder. Same thing here.

File: ubuntu_15.04.sh (hunk 1 above)
Comment (gonuke) [inline ubuntu_15.04.sh]: @Shimwell - did you have plans to replace the `rm -f` with a warning/error msg and halt?  I'd prefer that solution, so I can make a separate PR if you can't get to it.

**Comment by Shimwell** [comment]
Hi there
I can add something like this if you like


read -p "Delete the existing pyne directory and all contents? (y/n) "
-n 1 -rif [[ $REPLY =~ ^[Yy]$ ]]then
    rm -rf pynefi

**Comment by gonuke** [comment]
Sure - go ahead and add that...

**Comment by gonuke** [comment]
And also for the moab directory...

**Comment by kkiesling** [comment]
@Shimwell it says there are conflicts with the existing file. Can you resolve those first please? Then we can merge this in. Thanks.

**Comment by kkiesling** [comment]
Thanks for the additions @Shimwell. Merging.

**Comment by Shimwell** [comment]
I guess it is time to take a look at the new Ubuntu 17.04

**Pull request merged**